### ISSUE TITLE
[github_actions] Missing signature issues with current builds

### ISSUE THREAD

**Issue by LukeShortCloud** [LukeShortCloud/winesapOS#798]
At the start of the winesapOS install, packages are consistently missing signature files when trying to run a system updatae. This problem also affects the `pacstrap` that happens afterwards. The overall build fails in less than 3 minutes because of this.

```
+ pacman -S -y -y -u --noconfirm
```
```
checking keyring...
checking package integrity...
error: util-linux-libs: missing required signature
error: xz: missing required signature
error: curl: missing required signature
error: libassuan: missing required signature
error: shadow: missing required signature
error: util-linux: missing required signature
error: failed to commit transaction (invalid or corrupted package)
Errors occurred, no packages were upgraded.
```

```
+ pacstrap -i /winesapos base base-devel wget --noconfirm
```
```
error: util-linux-libs: missing required signature
error: xz: missing required signature
error: shadow: missing required signature
error: util-linux: missing required signature
error: curl: missing required signature
error: libassuan: missing required signature
error: failed to commit transaction (invalid or corrupted package)
Errors occurred, no packages were upgraded.
==> ERROR: Failed to install packages to new root
```

https://github.com/LukeShortCloud/winesapOS/actions/runs/8476943223/job/23227162446

**Comment by LukeShortCloud** [comment]
I tried updating the `archlinux-keyring` package during the install but ran into this new error:

```
:: Proceed with installation? [Y/n] 
:: Retrieving packages...
 archlinux-keyring-20240313-1-any downloading...
checking keyring...
checking package integrity...
loading package files...
checking for file conflicts...
:: Processing package changes...
upgrading archlinux-keyring...
==> ERROR: There is no secret key available to sign with.
==> Use 'pacman-key --init' to generate a default secret key.
error: command failed to execute correctly
```

It appears that the latest Arch Linux container update from 2 weeks ago completely removed the keyrings.

**Comment by LukeShortCloud** [comment]
I'm still running into issues.

I found this thread mentioning issues when using `wget` with Pacman. I have never ran into this problem and we've been using `wget` for years now.

https://bbs.archlinux.org/viewtopic.php?id=292070

The problem has been fixed and merged into Pacman 6.1.0. Perhaps the fix is actually causing problems?

https://gitlab.archlinux.org/pacman/pacman/-/merge_requests/79
